examples/client_test.cpp

```diff
@@ -1728,16 +1728,16 @@ COLUMN OPTIONS
 				for (lt::announce_entry const& ae : h.trackers())
 				{
 					auto best_ae = std::min_element(ae.endpoints.begin(), ae.endpoints.end()
-						, [](lt::announce_endpoint const& l, lt::announce_endpoint const& r) { return l.fails < r.fails; } );
+						, [](lt::announce_endpoint const& l, lt::announce_endpoint const& r) { return l.info_hashes[1].fails < r.info_hashes[1].fails; } );
```

Comment:
it's not obvious to me why info_hash 1 is used here. Is `info_hashes` guaranteed to always have 2 elements?

Reply:
Yes, `info_hashes` has a fixed size of 2 with with first entry being for protocol v1 and the second v2.

Reply:
so, for a v1 torrent, everything under `info_hashes[1]` will be cleared, right? so this `min_element` won't do anything for v1 torrents, right?

Reply:
Right. I'm not sure why index 1 is hard coded here. At the very least it should be picking the index based on the best protocol version supported by the torrent.